src/main/java/frc/robot/Auto.java

```diff
@@ -227,4 +219,33 @@ private CommandBase timingWait() {
         return new FunctionalWaitCommand(() -> 0.25);
     }
 
+    // Square auto using the drive relative command
+    public CommandBase squareAutoDriveRelative() {
+        return sequence(
+                drive.setGyro180(),
+                drive.driveRelative(new Translation2d(0, 0), 0, 3),
```

Comment:
I believe the `Translation2d` here is the coordinates to move to relative the current position. The second argument is the angle the robot should be facing when it gets there. So these commands will just cause the robot to spin around.

Reply:
Ah, alright, I will update this with coordinates there.